lib/APPReinforcementLearning/src/App.cpp

```diff
@@ -137,10 +188,89 @@ void App::loop()
  * Private Methods
  *****************************************************************************/
 
+void App::sendLineSensorsData() const
+{
+    ILineSensors&   lineSensors      = Board::getInstance().getLineSensors();
+    uint8_t         maxLineSensors   = lineSensors.getNumLineSensors();
+    const uint16_t* lineSensorValues = lineSensors.getSensorValues();
+    uint8_t         lineSensorIdx    = 0U;
+    LineSensorData  payload;
+
+    if (LINE_SENSOR_CHANNEL_DLC == maxLineSensors * sizeof(uint16_t))
+    {
+        while (maxLineSensors > lineSensorIdx)
+        {
+            payload.lineSensorData[lineSensorIdx] = lineSensorValues[lineSensorIdx];
+
+            ++lineSensorIdx;
+        }
+    }
+
+    /* Ignoring return value, as error handling is not available. */
+    (void)m_smpServer.sendData(m_serialMuxProtChannelIdLineSensors, &payload, sizeof(payload));
```

Comment:
Are you sure you do not want to handle the errors? What happens if the messages are not sent?